finch-core/src/main/scala/io/finch/Endpoint.scala

```diff
@@ -28,12 +28,13 @@ import com.twitter.finagle.httpx.path.Path
 import com.twitter.finagle.httpx.service.NotFoundService
 
 /**
- * A REST API endpoint that primary defines a ''route'' and might be converted
- * into a finagled service with ''toService'' method.
+ * A REST API endpoint that primary defines a ''route'' and might be
+ * implicitly converted into Finagle service ''Service[Req, Rep]''.
```

Comment:
I think it's better to describe that it requires an implicit `Route` to be converted to a Service